### PR TITLE
2nd task of Exceptions

### PR DESCRIPTION
**Pull request by IvayloTsvetkov**

### CODE REVIEW

File: Exceptions_2/src/com/company/Main.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+public class Main {
+
+    public static void main(String[] args) {
+        Range rn1 = new Range(0, 100);
+        Range rn2 = new Range(10, 20);
+        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+        try {
+            Integer n = new Integer(br.readLine());
+            rn1.rangeCheck(n);
```
Comment (mgenov) [inline Exceptions_2/src/com/company/Main.java]: range.rangeCheck is looking like naming duplication 

startRange.contains(n)  looks like better usage.

File: Exceptions_2/src/com/company/Range.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
+package com.company;
+
+/**
+ * Created by clouway on 11.07.17.
+ */
+public class Range {
+    private int min, max;
```
Comment (mgenov) [inline Exceptions_2/src/com/company/Range.java]: range has lower and higher boundaries. Min/Max looks like out of the context.

**Comment by IvayloTsvetkov** [comment]
done.

**Comment by mgenov** [comment]
You have to squash your commits before this PR to be merged.

**Comment by IvayloTsvetkov** [comment]
I think its done.

File: Exceptions_2/src/com/company/Range.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        this.upperBound = upperBound;
+    }
+
+    /**
+     *
+     * @param n gets integer and check if its in range
+     */
+    void contains(int n) {
+        if(n < lowerBound || n > upperBound) throw new OutOfRange(lowerBound, upperBound);
+        System.out.println("In range of " + lowerBound + " - " + upperBound + ".");
```
Comment (mgenov) [inline Exceptions_2/src/com/company/Range.java]: boolean contains(int n) 

The system.out statements should be out of the production code.

**Comment by IvayloTsvetkov** [comment]
done

**Comment by IvayloTsvetkov** [comment]
done

**Comment by IvayloTsvetkov** [comment]
done

**Pull request merged**